branding_company/model/sale_order_invoice_wizard.py

```diff
@@ -28,23 +11,20 @@ class SaleOrderInvoiceWizard(models.Model):
     """
     _inherit = "sale.advance.payment.inv"
 
-    def create_invoices(self, cr, uid, ids, context=None):
+    @api.multi
+    def create_invoices(self):
         """Add branding_id to new invoices created."""
-        res = super(SaleOrderInvoiceWizard, self).create_invoices(
-            cr, uid, ids, context=context)
-        sale_ids = context.get('active_ids', [])
-        if sale_ids:
-            # Go through all active sale-orders.
-            sale_model = self.pool['sale.order']
-            for sale_obj in sale_model.browse(
-                    cr, uid, sale_ids, context=context):
-                if sale_obj.branding_id:
-                    for invoice_obj in sale_obj.invoice_ids:
-                        # Fill branding_id in invoices that don't
-                        # have one yet.
-                        if not invoice_obj.branding_id:
-                            invoice_obj.write(
-                                {'branding_id': sale_obj.branding_id.id},
-                                context=context
-                            )
+        res = super(SaleOrderInvoiceWizard, self).create_invoices()
+        sale_ids = self.env.context.get('active_ids', [])
+        if not sale_ids:
+            return
+        # Go through all active sale-orders.
+        for sale_obj in self.env['sale.order'].browse(sale_ids):
+            if sale_obj.branding_id:
+                for invoice_obj in sale_obj.invoice_ids:
```

Comment:
you can save some levels of indentation by using `.filtered()`, `.mapped`, but I won't block on this